lib/tools/adb-commands.js

```diff
@@ -400,19 +404,31 @@ methods.toggleGPSLocationProvider = async function toggleGPSLocationProvider (en
  *        or to a restricted greylist for your target API level.
  */
 methods.setHiddenApiPolicy = async function setHiddenApiPolicy (value) {
```

Comment:
it would probably be more performant to catenate all commands into one and separate them with `;`. Something we do in grantPermissions helper